Lifestyle/ZoomsToday/zooms-today.1h.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+#
+# Display a list of Zoom based meetings in your calendar today.
+# Selecting a meeting will take you to the Zoom session associated with it.
+#
+# <bitbar.title>Zoom Meetings for Today</bitbar.title>
+# <bitbar.version>1.0.0</bitbar.version>
+# <bitbar.author>Luis Cruz</bitbar.author>
+# <bitbar.author.github>sprak3000</bitbar.author.github>
+# <bitbar.desc>Display a list of Zoom based meetings in your calendar today. Selecting a meeting will take you to the Zoom session associated with it.</bitbar.desc>
+# <bitbar.dependencies>icalBuddy</bitbar.dependencies>
```

Comment:
Would it be possible to get an image here?

Reply:
Done!